<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WHAT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 9pt &quot;Arial&quot;;
border: 2px dotted #000000; border-radius: 10px;

</string>
    </property>
    <property name="title">
     <string>Chrome</string>
    </property>
    <widget class="QGroupBox" name="verticalGroupBox_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>39</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:None;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string> Cache</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string> Cookie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string> History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string> Download List</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial&quot;;
border: 2px dotted #000000; border-radius: 10px;

</string>
    </property>
    <property name="title">
     <string>Internet Explorer</string>
    </property>
    <widget class="QGroupBox" name="verticalGroupBox_5">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>39</y>
       <width>171</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:None;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string> Cache</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="text">
         <string> Cookie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="text">
         <string> History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_8">
        <property name="text">
         <string> Download List</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>561</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <column>
         <property name="text">
          <string>url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>메타데이터 크기</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>데이터 크기</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>메타데이터 or 캐시 파일명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>데이터 존재여부 or 캐시 파일명</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_2">
        <column>
         <property name="text">
          <string>쿠키 이름</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>쿠키 생성 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>쿠키 만료 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 접근 시간</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_3">
        <column>
         <property name="text">
          <string>url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>웹페이지 제목</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>방문 횟수</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 방문 시간</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_4">
        <column>
         <property name="text">
          <string>현재 경로</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>다운로드 데이터 크기</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 접근 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 수정 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>소스 url</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>561</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>561</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTreeWidget" name="ieCache">
        <column>
         <property name="text">
          <string>마지막 수정 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 접속 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>캐시 디렉터리 인덱스</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>캐시 파일명</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>http 헤더</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="ieCookie">
        <column>
         <property name="text">
          <string>마지막 수정 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 접속 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>쿠키 파일명</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="ieHistory">
        <column>
         <property name="text">
          <string>url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>방문 횟수</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 수정 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>마지막 방문 시간</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="ieDownloadList">
        <column>
         <property name="text">
          <string>마지막 접속 시간</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>다운로드 데이터 크기</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>소스 url</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>저장 경로</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;돋움체&quot;;
background-color: #d9d9df</string>
    </property>
    <property name="text">
     <string>닫기</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 244, 244);</string>
   </property>
   <widget class="QMenu" name="menuBrowser">
    <property name="title">
     <string>브라우저</string>
    </property>
    <addaction name="actionChrome"/>
    <addaction name="actionInternet_Explorer"/>
   </widget>
   <widget class="QMenu" name="menufunction">
    <property name="title">
     <string>기능</string>
    </property>
    <addaction name="actionstatistic"/>
    <addaction name="actionexcel"/>
   </widget>
   <addaction name="menuBrowser"/>
   <addaction name="menufunction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChrome">
   <property name="text">
    <string>Chrome</string>
   </property>
  </action>
  <action name="actionInternet_Explorer">
   <property name="text">
    <string>Internet Explorer</string>
   </property>
  </action>
  <action name="actionstatistic">
   <property name="text">
    <string>통계</string>
   </property>
  </action>
  <action name="actionexcel">
   <property name="text">
    <string>엑셀 추출</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
